src/libmongoc/tests/test-mongoc-client-side-encryption.c

```diff
@@ -680,6 +682,174 @@ test_datakey_and_double_encryption_creating_and_using (
    mongoc_client_encryption_datakey_opts_destroy (opts);
 }
 
+/* Prose test "Create key with custom key material" */
```

Comment:
This should probably be numbered. See: https://github.com/mongodb/specifications/pull/1213/files#r875104251